File: Domain/Project.cs
using System;
using System.Reflection.Metadata;

namespace Domain
{
    public class Project
    {
        public string Id { get; set; }
        public string Titel { get; set; }
        public string Description { get; set; }

        public DateTime StartDate { get; set; }

        public DateTime EndDate { get; set; }

        public string CurrentStatus { get; set; }

        public int TotalProjectDays { get; set; }
        public Contact Contact { get; set; }
    }
}

File: client-app/src/app/models/project.ts
import { IContact } from "./contact";
import { IEmployee } from "./employee";

export interface IProject{

    id:string;
    titel:string; 
    description?:string; 
    startDate?: string;
    endDate?: string;
    currentStatus?: string; 
    totalProjectDays?: number;
    contact?: IContact;
    projectLeader?: IEmployee
    
}